openmls/src/utils.rs

```diff
@@ -1,15 +1,11 @@
 use crate::extensions::*;
 use crate::tree::{index::*, node::*, *};
 
-use evercrypt::prelude::*;
+// === The folowing functions aren't necessarily cryptographically secure!
```

Comment:
Which should be ok, because they're behind the test-utils or test config flags.

Reply:
right, they are probably fine. But I just want to make sure everyone knows that we no one made sure that these are safe.